testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/autoignore/AutoIgnoredResourcesDomainTestCase.java

```diff
@@ -493,7 +493,6 @@ public void test51_testCompositeOperation() throws Exception {
 
         steps.add(addGroupOp);
         steps.add(Util.getWriteAttributeOperation(getSlaveServerConfigAddress(SERVER1), SOCKET_BINDING_GROUP, SOCKETS3));
-        steps.add(Util.getWriteAttributeOperation(getSlaveServerConfigAddress(SERVER1), GROUP, GROUP2));
```

Comment:
I don't think we want to change this test, as it is testing some very hard use cases and the failure is likely telling us there is something wrong with this fix.

I suspect that what it's telling us is that the validation step you've added needs to handle the fact that the socket binding group may no longer exist by the time that step executes. It handles it by returning cleanly; if the group is no longer present, then the child cannot be invalid.